lib/fileUpload.coffee

```diff
@@ -13,8 +13,8 @@ if UploadFS?
 		cookie = new Cookies()
 		if Meteor.isClient
 			document.cookie = 'rc_uid=' + escape(Meteor.userId()) + '; path=/'
-			document.cookie = 'rc_token=' + escape(Meteor._localStorage.getItem('Meteor.loginToken')) + '; path=/'
-
+			document.cookie = 'rc_token=' + Accounts._storedLoginToken()
```

Comment:
I'm unsure of the functionality of escape. Is it the (deprecated) javascript escape function, underscore.js's escape function, or something else? Either way it seems right to follow the format of what was there before :) (though it didn't seem to make a difference for me).